TaskBroker/VstsServerTaskBroker.Contracts/VstsServerTaskBroker.Contracts.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <ProjectGuid>{E40964B3-3B6A-48F2-8A3A-285C4CA2FFDB}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>VstsServerTaskBroker.Contracts</RootNamespace>
+    <AssemblyName>VstsServerTaskBroker.Contracts</AssemblyName>
+    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
```

Comment:
Moved these libraries to use .Net Standard (.NET core). Had to rework Azure.ServiceBus as the .NET core version has different interface.